client/app/lib/components/core/BarChart.jsx

```diff
@@ -55,4 +57,4 @@ BarChart.propTypes = {
   ).isRequired,
 };
 
-export default BarChart;
+export default memo(BarChart, equal);
```

Comment:
This is needed? I remember you removed it in the previous PR

Reply:
this is weird. I think when I did the rebasing, this line got reverted back (without me realising it). I should next time check the files that are changed and no conflict as well.

Thus far, I resolved the conflict only but not recognising the files that are changed with no conflict -> this might be the reason why I didn't notice this line being reverted back